YubiKit/YubiKitTests/Tests/YKFOATHCredentialTests.m

```diff
@@ -204,6 +205,213 @@ - (void)test_WhenCredentialIsCreatedWithTOTPURLWithCustomPeriod_KeyContainsThePe
     XCTAssert([credential.key isEqualToString:expectedKey], @"Credential key for TOTP with custom period does not contain the period.");
 }
 
+- (void)test_WhenKeyContainsSlashAndPeriodExists_PeriodIsParsed {
```

Comment:
Can we extract all of this tests into separate file, they seems to test YKNSStringAddition method rather than YKFOATHCredential